### ISSUE TITLE
Integrations.Search.Algolia - build errors / error on installation in Umbraco v11.0.0

### ISSUE THREAD

**Issue by d-gibbs** [umbraco/Umbraco.Cms.Integrations#73]
Hi,

I've installed the `Umbraco.Cms.Integrations.Search.Algolia` package in a new Umbraco v11.0.0 project and am facing an issue with installation/build.

Upon install, it did not automatically pull down its dependencies e.g.

- Umbraco.Cms.Web.BackOffice
- Umbraco.Cms.Web.Common
- Umbraco.Cms.Web.Website

So it failed on initial build resulting in the following error:

![image](https://user-images.githubusercontent.com/59481722/211340206-7f4981da-b089-4bef-a4dd-cd849d06f134.png)

To resolve this I manually added the above dependencies which seemed to fix the problem however we're still seeing warnings relating to the package version during build:

```
1>warning NU1608: Detected package version outside of dependency constraint: Umbraco.Cms.Integrations.Search.Algolia 1.0.0 requires Umbraco.Cms.Web.BackOffice (>= 10.3.0 && < 11.0.0) but version Umbraco.Cms.Web.BackOffice 11.0.0 was resolved.
1>warning NU1608: Detected package version outside of dependency constraint: Umbraco.Cms.Integrations.Search.Algolia 1.0.0 requires Umbraco.Cms.Web.Common (>= 10.3.0 && < 11.0.0) but version Umbraco.Cms.Web.Common 11.0.0 was resolved.
1>warning NU1608: Detected package version outside of dependency constraint: Umbraco.Cms.Integrations.Search.Algolia 1.0.0 requires Umbraco.Cms.Web.Website (>= 10.3.0 && < 11.0.0) but version Umbraco.Cms.Web.Website 11.0.0 was resolved.
```

Could this be an issue with the package version constraint? Should it be `>= 10.3.0` without the upper bound instead?

Thank you

**Comment by acoumb** [comment]
Hi @d-gibbs 
I've released version [1.1.0](https://www.nuget.org/packages/Umbraco.Cms.Integrations.Search.Algolia/1.1.0) of Algolia with a modified upper bound.
I will close this now, feel free to reach out if you have other input.
Thank you!

**Comment by d-gibbs** [comment]
That's done the trick, thanks for sorting this!
